plugins/crypto/mbedtls/ua_securitypolicy_aes256sha256rsapss.c

```diff
@@ -734,6 +735,144 @@ updateCertificateAndPrivateKey_sp_aes256sha256rsapss(UA_SecurityPolicy *security
     return retval;
 }
 
+static UA_StatusCode
```

Comment:
Is this amount of code duplication really necessary? Some common parts for mbedTLS are already factored out into the common source file. So why not this too?